packages/frontend/src/lib/table/service/ServiceAction.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+import type { InferenceServer } from '@shared/src/models/IInference';
+import { studioClient } from '/@/utils/client';
+import { faPlay, faStop, faTrash } from '@fortawesome/free-solid-svg-icons';
+import ListItemButtonIcon from '/@/lib/button/ListItemButtonIcon.svelte';
+export let object: InferenceServer;
+
+function stopInferenceServer() {
+  studioClient.stopInferenceServer(object.container.containerId);
+}
+
+function startInferenceServer() {
+  studioClient.startInferenceServer(object.container.containerId);
```

Comment:
Should we display the error if the start fails?

Reply:
We are not doing it inside the Environment page, so I do not think so ? Where would you want it to be displayed ?

Reply:
I added a catch block so at least for now we do not lost it